dspace-api/src/test/java/org/dspace/AbstractIntegrationTestWithDatabase.java

```diff
@@ -105,7 +105,7 @@ public static void initDatabase() {
     public void setUp() throws Exception {
         try {
             //Start a new context
-            context = new Context(Context.Mode.BATCH_EDIT);
+            context = new Context(Context.Mode.READ_WRITE);
```

Comment:
READ_WRITE was actually the mode used even previously (the one set here was immediately overwritten)